esbuildConfig.js

```diff
@@ -0,0 +1,128 @@
+const { sassPlugin, postcssModules } = require('esbuild-sass-plugin');
+const autoprefixer = require('autoprefixer');
+const postcss = require('postcss');
+const yargsParser = require('yargs-parser');
+const { writeFileSync } = require('fs');
+const { resolve, join } = require('path');
+const alias = require('esbuild-plugin-alias');
+
+// from https://github.com/bvaughn/react-virtualized/issues/1212#issuecomment-847759202 workaround for https://github.com/bvaughn/react-virtualized/issues/1632 until it is released.
+const resolveFixup = {
+    name: 'resolve-fixup',
+    setup(build) {
+        build.onResolve({ filter: /react-virtualized/ }, async (args) => {
+            return {
+                path: resolve(join(__dirname, './packages/ui-components/node_modules/react-virtualized/dist/umd/react-virtualized.js'))
```

Comment:
That looks error prone. Wouldn't it be better to use something like `join(require.resolve('react-virtualized'), 'dist/umd/react-virtualized.js')`